source/index.rst

```diff
@@ -5,49 +5,27 @@
 
 .. include:: ../README.rst
 
-Contents
-========
-This wiki is split into four sections, namely: `General`_, `Server Admins`_, `Players`_, and `Developers`_.
+.. _home:
 
-General
---------------
+This wiki is a work in progress, if you notice any issue or error, you are welcome to report it on our `GitHub`_
 
-This section contains general information thats relative to everyone
 
 .. toctree::
-    :maxdepth: 2
+    :maxdepth: 1
+    :caption: General
+    :name: general
 
-    general/index
-    general/commands
-    general/faq
-    general/pkcore
+    Home <general/index>
+    FAQ <general/faq>
+    Contributing to the Wiki <general/wikicontribute>
 
-Server Admins
---------------
-
-This section will contain pages that will help server admins use and configure ProjectKorra.
-
-.. toctree::
-    :maxdepth: 2
-
-    admins/index
-
-Players
---------------
-
-This section will help players with commands and general gameplay with the plugin.
 
 .. toctree::
-    :maxdepth: 2
-
-    players/index
-
-Developers
---------------
-
-This section help developers understand ProjectKorra and contain development tutorials.
-
-.. toctree::
-    :maxdepth: 2
-
-    developers/index
+    :maxdepth: 1
+    :caption: Core
```

Comment:
ProjectKorra Core / Core Plugin would be a better caption

Reply:
I prefer the names to be short.
The first version looks better imo
CORE --> PROJECTKORRA CORE
RPG --> PROJECTKORRA RPG
ITEMS --> PROJECTKORRA ITEMS